presentation-api/index.html

```diff
@@ -29,6 +29,10 @@
   <button id="reconnect" disabled>presentationRequest.reconnect(presentationId)</button>
 </p>
 <p>
+  <input id="message" type="text" placeholder="Try a fruit!">
```

Comment:
`"list=fruits"` does not force user to pick a fruit, it is a simple suggestion list.
Therefore, I'd suggest we remove "sendMessage" there.